share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -531,6 +531,7 @@ sxc,sexcoin,sex coin,
 sha,shacoin,sha coin,
 shade,shadecoin,shade coin,
 sdc,shadowcoin,shadow coin,
+sdc, shadowcash, shadow cash
```

Comment:
Would these extra triggers cause a bug?
The name has changed, thus I thought it would be a good idea to add the extra different ones.